study.md

```diff
@@ -66,15 +72,20 @@ nearest_star = the_sun
 Will modifying `sol` affect `the_sun`? Why or why not?
 
 ```md
-<!-- your answer here -->
+Modifying 'sol' will not affext 'the_sun' because it is a seperate class instance
+of Star. Both instances were made independently from the Star parent class and
+therefore modifying either will not affect the other.
 ```
 
 ## Ruby Object Equality: Part II
 
 Will modifying `the_sun` affect `nearest_star`? Why or why not?
 
 ```md
-<!-- your answer here -->
```

Comment:
Modifications to `the_sun` or `nearest_star` will affect the other because both variables are referencing the same object.